packages/libs/eda/src/lib/workspace/AnalysisPanel.tsx

```diff
@@ -143,10 +144,24 @@ export function AnalysisPanel({
 
   const [lastVarPath, setLastVarPath] = useState('');
   const [lastVizPath, setLastVizPath] = useState('');
-  const [globalFiltersDialogOpen, setGlobalFiltersDialogOpen] = useState(false);
-  const [sharingModalVisible, setSharingModalVisible] = useState<boolean>(
-    false
-  );
+  // const [globalFiltersDialogOpen, setGlobalFiltersDialogOpen] = useState(false);
+  const [sharingModalVisible, setSharingModalVisible] =
+    useState<boolean>(false);
+
+  /**
+   * Used in FilterChipList to highlight filter chip if/when an on-view Browse & Subset
+   * var matches a filter chip. Logic is:
+   * 1. check pathname to determine if Browse & Subset tab is selected
+   * 2. split lastVarPath into an array structured as ['', entityId, variableId]
+   */
+  const isSubsetTabSelected = location.pathname.includes('variables');
+  const splitLastVarPath = lastVarPath.split('/');
```

Comment:
**nit**: we could prolly just call this `lastVarPath` without referencing `.split()`

Reply:
Can you please elaborate on this? Initially I didn't declare a new variable (`splitLastVarPath`) and instead did something like `lastVarPath.split('/')[1]`. Since I did it twice, I figured I'd just go ahead and declare it as a variable. I'm not quite understanding how we'd "just call this `lastVarPath`" since that variable name already exists in the namespace.

Reply:
Ah yes, that's totally a thing that I overlooked.

Maybe something like `lastVarPathParts` or `lastVarPathList`? I was poorly trying to say express that "split" feels verb-y, but that the rest of the variable name is composed of nouns. Moreover, if we somehow change `lastVarPath` from a string to a list of string (doubt we'd do that), then `split` would be confusing.

That all said, this isn't at all a big deal 🙃

Reply:
Ah, yes. My mind was reading "split" as a noun (like banana split), but that's not at all intuitive! I'll change its name so it doesn't read as a function name (verb).

Reply:
Wow, I forgot about banana splits! 

> A banana split is an American ice cream-based dessert consisting of a peeled banana cut in half lengthwise, and served with ice-cream and sauce between the two pieces. There are many variations, but the classic banana split is made with three scoops of ice cream (one each of vanilla, chocolate, and strawberry).
> \- https://en.wikipedia.org/wiki/Banana_split